templates/default/delos.css

```diff
@@ -5910,7 +5910,7 @@ button.close {
   height: 10px;
   margin: 1px;
   text-indent: -999px;
-  border: 1px solid #ffffff;
+  border: 1px solid #fff;
```

Comment:
Isn't that part of the installer or build process problem? It would be clean to only commit less and no css and let the client side builder/installer do the rest, since we do not really want to care about that via git. Similar to the composer issue. For the time being a believe to best option is the build the css and commit and live with the noise this process is accompanied by. The other option would be the picking of actually changed lines. I would hate that, since this would lead to quite a source of errors to to missing affected lines. Think about what you are asking for, this would have to be done with each less change each time a variable is changed. If you change an important one, you would have to manually pick up to a few hundred css lines manually! If you miss one, you break the CSS.